daemon/volumes.go

```diff
@@ -17,8 +17,7 @@ import (
 // a volume mount that is not writable.
 var ErrVolumeReadonly = errors.New("mounted volume is marked read-only")
 
-// TODO Windows. Further platform refactoring can still be done in volumes*.go
-
+// mountPoint holds the information pertaining to a mount point.
```

Comment:
mountPoint is the intersection point between a Volume and a container. It specifies which volume is to be used and where Docker should mount it to inside a container.

Feel free to clean that up... but as is the doc is tautological.